lib/sufia/arkivo/schema_validator.rb

```diff
@@ -0,0 +1,57 @@
+require 'json-schema'
+
+module Sufia
+  module Arkivo
+    ITEM_SCHEMA = {
+      type: 'object',
+      properties: {
+        token: { type: 'string', required: true },
+        metadata: {
+          type: 'object',
+          required: true,
+          properties: {
+            title: { type: 'string', required: true },
+            creators: { type: 'array', required: true },
+            rights: { type: 'string', required: true },
+            tags: { type: 'array', required: true },
```

Comment:
@mjgiarlo by default Rails converts empty arrays in a JSON request to `nil` (see [here](https://github.com/rails/rails/issues/13766)) — with 4.1 and later there is a way to disable this, but I guess the easier solution is to permit nil values for `tags` and `creators` since the empty array sent by Arkivo arrives as nil. What do you think?